projects/unit/configs/all/shared_dec.yaml

```diff
@@ -0,0 +1,193 @@
+model_config:
+  unit:
+    base_args:
+      num_queries:
+        detection:
+          detection_coco: 100
+          detection_visual_genome: 100
+        vl:
+          vqa2: 25
+          # hateful_memes: 25
+          visual_entailment: 25
+        glue:
+          glue_qnli: 25
+          glue_qqp: 25
+          glue_sst2: 25
+          glue_mnli_mismatched: 25
+      share_decoders: true
+      decoder_hidden_dim: 768
+      dilation: true
+      use_task_embedding_in_img_encoder: true
+      use_task_embedding_in_lang_encoder: true
+    bert_config:
+      bert_model_name: bert-base-uncased
+    losses:
+    - logit_bce
+    # initialize the ResNet convnet backbone from DETR
+    base_ckpt_path: https://dl.fbaipublicfiles.com/detr/detr-r50-dc5-f0fb7ef5.pth
+    base_ckpt_load_backbone_only: true
+
+dataset_config:
+  vqa2:
+    zoo_requirements:
+    - coco.defaults
+    - vqa2.defaults
+    - vqa2.split_by_coco_2017
+    return_features_info: true
+    use_features: false
+    use_images: true
+    processors:
+      text_processor:
+        type: bert_tokenizer
+        params:
+          tokenizer_config:
+            type: bert-base-uncased
+            params:
+              do_lower_case: true
+          mask_probability: 0
+          max_seq_length: 25
+      image_processor:
+        type: torchvision_transforms
+        params:
+          transforms:
+            - type: ResizeShortest
+              params:
+                min_size: 800
+                max_size: 1333
+            - ToTensor
+            - type: Normalize
+              params:
+                mean: [0.485, 0.456, 0.406]
+                std: [0.229, 0.224, 0.225]
+    images:
+      train:
+      - coco/defaults/images/trainval2014
+      - coco/defaults/images/trainval2014
+      - coco/defaults/images/trainval2014
```

Comment:
COCO 2014 usually doesn't have a single `trainval2014` directory. `train2014` and `val2014` are separate directories. Please make sure this is what is extracted from the zoo downloads or else users will find it difficult to train this model.

Reply:
(split the imdbs w.r.t. train2014 and val2014)